### ISSUE TITLE
Spells page could use some clarity on how tiers work

### ISSUE THREAD

**Issue by nyanlathotep** [G00FBALL/not-a-wiki#26]
The spell tiers section should have some explanation of how the spells' bonuses compound with tiers, the exception for gem grinder, and some note that peripheral effects like fairy choir are not increased.
It might also be worthwhile to note that some tiers have extra effects.

**Comment by G00FBALL** [comment]
not sure how to do this without being confusing to me or other players

**Comment by Alfik0** [comment]
Formulas if someone want to do it
```
newM=oldM^(T*0.1) where 
newM is new(post tiers) multiplier, 
oldM is old(pre tiers) multiplier, 
T is spell tier
```
```
newB=100*((oldB/100+1)^(T*0.1)-1) where
newB is new(post tiers) production bonus,
oldB is old(pre tiers) production bonus,
T is spell tier
```

**Comment by schaefer-austin-p** [comment]
Can we close this? PR #56 was merged a bit back.

**Comment by nyanlathotep** [comment]
Oh sure, and for future reference, you can always toss a "closes #26" or similar in a commit message to auto-close an issue when a commit made/merged.
